.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
I am disabling `duplicate-code` in pylintrc as well. It says that [this](https://github.com/oppia/oppia-ml/pull/10/files#diff-38cf87a4ad2714386a39b68573d3cca2R52) and [this](https://github.com/oppia/oppia-ml/pull/10/files#diff-ed3af7853d03776aecf2b00cbb956c1dR41) line have similar code and they are designed in such way. I can't disable it using inline comment (I tried adding comment but it didn't work).